Userland/Libraries/LibSQL/Heap.cpp

```diff
@@ -143,32 +186,33 @@ void Heap::flush()
             VERIFY_NOT_REACHED();
         }
         dbgln_if(SQL_DEBUG, "Flushing block {} to {}", block, name());
-        write_block(block, buffer_or_empty.value());
+        TRY(write_block(block, buffer_or_empty.value()));
     }
     m_write_ahead_log.clear();
     dbgln_if(SQL_DEBUG, "WAL flushed. Heap size = {}", size());
+    return {};
 }
 
-constexpr static const char* FILE_ID = "SerenitySQL ";
+#define SERENITY_SQL "SerenitySQL"
+constexpr static StringView FILE_ID(SERENITY_SQL " "sv);
```

Comment:
```c++
constexpr static StringView FILE_ID = "SerenitySQL  "sv;
constexpr static StringView SERENITY_SQL = FILE_ID.substring_view(0, FILE_ID.length() - 1);
```
Though also, what's the space at the end of `FILE_ID` actually for?

Reply:
I also think I was overthinking this. I thought it would be useful to have the string `"SerenitySQL"` around, but I'm not using it anywhere except in the `FILE_ID` and I also can't think where it could be needed.